docs/finalproject/roles.md

```diff
@@ -11,39 +11,33 @@ These roles should in total create a [balanced team](https://www.youtube.com/wat
 ## Volunteer Roles
 
 ### Product Manager // Scrum Master
+Full role description is [here](https://docs.codeyourfuture.io/volunteers/teams-1/cyf-products-final-projects/roles/product-manager)
 
-**Time Commitment**: One day a week plus 2-4 hours during the week
-
-Scrum Masters are expected to coordinate the team and make sure everyone knows what they are working on. They can provide support in
-
+Your main role is to coordinate the team and make sure everyone knows what they are working on. They can provide support in:
 - Scheduling meetings
-- Working with the team to create and update tasks on Trello
+- Working with the team to create and update tasks on Github Projects
 - Introducing and enforcing Agile practices (e.g. sprint planning, retros)
 
-### Technical Assistant
-
-**Time Commitment**: One day a week plus 2-4 hours during the week
-
-Teaching assistant provide technical support to the trainees as they are developing their projects. They can provide support in
+### Technical Lead
+Full role description and commitment is [here](https://docs.codeyourfuture.io/volunteers/teams-1/cyf-products-final-projects/roles/tech-lead)
 
+The TL provide technical support to the trainees as they are developing their projects. They can provide support in:
 - Architecture of the application
 - Helping to solve specific implementation problems
 - Helping to solve specific bugs or errors
 - Providing overview of the entire project
 
 ### UI/UX Designer
```

Comment:
```suggestion
### User Interface/ User Experience (UI/UX) Designer
```
Acronyms should be introduced once before use. https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/G102